spec/unit/inflecto/class_methods/uncountable_predicate_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Inflecto, '.uncountable?' do
+  it 'truthy when empty string' do
+    Inflecto.uncountable?('').should == true
+  end
+
+  it 'truthy when word is present in list' do
+    Inflecto.uncountable?('grass').should == true
+  end
+
+  it 'falsy when word is not present in list' do
+    Inflecto.uncountable?('user').should == false
+  end
+
+  it 'truthy when word is present in list but in different case' do
+    Inflecto.uncountable?('FiSH').should == true
```

Comment:
See https://github.com/mbj/inflecto/pull/8/files#r22766616

Also `FiSH` might be a typo of `Fish` ?